.travis.yml

```diff
@@ -1,15 +1,37 @@
 language: php
 
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
```

Comment:
I suggest caching only the ``files`` subdolder. See https://github.com/puli/manager/pull/9#discussion_r40499480 for the explanation